src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/view-index-top.jelly

```diff
@@ -24,36 +24,39 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:this="this">
-  <j:set var="iconClass" value="${it.iconColor.getIconClassName()}"/>
-  <j:if test="${iconClass != null}">
-      <j:set var="iconClass" value="${iconClass} icon-xlg"/>
-  </j:if>
-  <h1>
+  <div class="jenkins-app-bar">
+    <div class="jenkins-app-bar__content jenkins-build-caption">
+      <j:set var="iconClass" value="${it.iconColor.getIconClassName()}"/>
+      <j:if test="${iconClass != null}">
+        <j:set var="iconClass" value="${iconClass} icon-lg"/>
+      </j:if>
+      <j:choose>
+        <j:when test="${iconClass != null}">
+          <l:icon class="${iconClass}" alt="${it.iconColor.description}" tooltip="${it.iconColor.description}" />
+        </j:when>
+        <j:otherwise>
+          <l:icon src="${it.iconColor.getImageOf('32x32')}" alt="${it.iconColor.description}" tooltip="${it.iconColor.description}" />
+        </j:otherwise>
+      </j:choose>
+      <h1 class="job-index-headline page-headline">
+        <l:breakable value="${it.displayName}"/>
+      </h1>
+    </div>
+    <div class="jenkins-app-bar__controls">
+      <t:editDescriptionButton description="${it.primaryView.description}" permission="${it.CONFIGURE}"/>
+    </div>
+  </div>
+  <j:if test="${(it.fullName!=it.fullDisplayName)}">
     <j:choose>
-      <j:when test="${iconClass != null}">
-        <l:icon class="${iconClass}" tooltip="${it.iconColor.description}" />
+      <j:when test="${it.parent!=app}">
+        ${%Full folder name}: ${it.fullName}
       </j:when>
       <j:otherwise>
-        <l:icon src="${it.iconColor.getImageOf('48x48')}" height="48" width="48" alt="${it.iconColor.description}" tooltip="${it.iconColor.description}" />
+        ${%Folder name}: ${it.fullName}
       </j:otherwise>
     </j:choose>
-    ${it.displayName}
-  </h1>
-  <j:if test="${it.name != it.displayName}">
-    ${%Folder name}: ${it.fullName}
   </j:if>
-  <div id="view-message">
-      <div id="systemmessage">
-          <j:out value="${app.systemMessage!=null ? app.markupFormatter.translate(app.systemMessage) : ''}"/>
-      </div>
-      <j:out value="${it.description!=null ? app.markupFormatter.translate(it.description) : ''}"/>
-  </div>
```

Comment:
this removes the system message from the folder.

## before

![image](https://github.com/user-attachments/assets/b6bb88fc-3def-4654-bd2c-ab98507b6d44)
![image](https://github.com/user-attachments/assets/55ba4977-6588-48e8-bd54-26ce413a4689)


## after
![image](https://github.com/user-attachments/assets/04b09539-0127-4c52-a148-2fbd529b5f96)
![image](https://github.com/user-attachments/assets/c6219d2b-d8a2-4a9f-b8d8-710d42de9ba2)

Reply:
It is not shown for Jobs, so I'm wondering, do we actually _want_ the system message to be displayed there?

Reply:
> It is not shown for Jobs, so I'm wondering, do we actually _want_ the system message to be displayed there?

I would argue we do want it for `Items` (and more..) (and it was never implemented¹).  
if an admin uses a system message to convey some "downtime" then users interacting with Jenkins would never see it unless they where on the root page (which is unlikely if they follow links from say GH checks or just go to thier job).

¹ I actually have a (likely false) memory that it did used to show up for Items also!

Reply:
I would suggest that the discussion about where the systemMessage should or should not be should happen outside this PR as it impacts usability over and above the other changes.